src/test/resources/roles.yml

```diff
@@ -1128,7 +1128,35 @@ data_stream_3:
         - "*"
       allowed_actions:
         - "DATASTREAM_ALL"
+point_in_time_1:
+  reserved: true
+  hidden: false
+  description: "Migrated from v6 (all types mapped)"
+  cluster_permissions:
+    - "cluster:admin/point_in_time/read_from_nodes"
```

Comment:
Without cluster level permission we'd not be able to get all PITs from the cluster, as the request doesn't have indices as part of it. Any suggestions ?

Reply:
Is there a place where all of the different Rest APIs are documented in the form below?  I would like to be able to cross-reference the actions and associated permissions that are needed to perform them, I think that might help our understanding.

#### Create point in time
```
POST "/<target indices>/point_in_time?*keep_alive*=1h&routing=&expand_wildcards=&preference=" 
```

Reply:
https://github.com/opensearch-project/OpenSearch/issues/3960 -- is the design doc

Reply:
The doc and the comments on this PR have evolved over time and it is not clear to me what the permission setup, user action, and result should be for the many scenarios embodied - please make this as clear as possible.

Not only would this help us understand the specific scope and bounds of your request, this would be of great value for writing the end-user facing permissions documentation for this feature